cdap-standalone/src/main/java/io/cdap/cdap/StandaloneMain.java

```diff
@@ -392,7 +390,7 @@ public void shutDown() {
     }
   }
 
-  private void cleanupTempDir() {
+  private static void cleanupTempDir(CConfiguration cConf) {
     File tmpDir = new File(cConf.get(Constants.CFG_LOCAL_DATA_DIR),
                            cConf.get(Constants.AppFabric.TEMP_DIR)).getAbsoluteFile();
 
```

Comment:
Construct the path at callsite and pass that in, so it is clear what state the function needs instead of passing the whole configuration structure that this function doesn't care about the other n-2 fields

Reply:
Fixed